estate/models/__init__.py

```diff
@@ -0,0 +1 @@
+from . import estate_property, estate_property_type, estate_property_tag, estate_property_offer
```

Comment:
```suggestion
from . import estate_property
from . import estate_property_type
from . import estate_property_tag
from . import estate_property_offer
```
I'm not sure why but we do it like this :smile: